pkg/processor/build/runtime/dotnetcore/docker/onbuild/Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# make the processor binary
+FROM nuclio/processor AS processor
+
+#*********************************************
+
+FROM microsoft/dotnet:2-sdk AS build-env
+
+# Get .NET core SDK 
+WORKDIR /app
+RUN apt-get update && \
```

Comment:
apt-get populates a local cache and that negatively affects the image size. The pattern we use is:
```
RUN apt-get update \ 
    && apt-get install -y <whatever> \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* \
    ...
```

On second thought in an effort to reduce size further, why not download the SDK via wget? Must we have git in the onbuild? 

Also, if you want (not required) - we can reduce the layers in this image by compacting a few commands / re-arranging order of commands).

Reply:
Done but few notes:
1. This is the onbuild. The processor is based on [`2-runtime`](https://github.com/torosent/nuclio/blob/1115a7469515c4870b4d2ee04961ebd7b6f30d35/pkg/processor/build/runtime/dotnetcore/runtime.go#L43)  which is 219 MB
2. The git in the onbuild is to fetch the latest SDK. I would like to have an official release of the .NET core SDK and just download it without git.